docs/content.md

```diff
@@ -1,6 +1,14 @@
 # Installation [installing]
 ## Installing the Replay Mod [replaymod]
-The **ReplayMod** requires **Minecraft Forge** to be installed.
+### Minecraft 1.14 and above
+The **ReplayMod** for Minecraft 1.14 and above requires **Fabric & Fabric API** to be installed.
+
+You can find Fabric and the installation instructions [here](https://fabricmc.net/use/). With Fabric, you also need to install the Fabric API by putting it into your mods folder. You can find Fabric API [here](https://www.curseforge.com/minecraft/mc-mods/fabric-api).
+
+After installing Fabric & Fabric API, simply put the downloaded ReplayMod .jar file into the `/mods` folder of your Minecraft directory.
+
+### Minecraft 1.12.2 and below
+For Minecraft 1.12.2 and below it requires **Minecraft Forge** to be installed.
 
 Depending on your **Replay Mod** version we recommend the following **Forge** versions:
```

Comment:
We should probably remove this section. It's incomplete and in general people should just use the most recent recommended build for their version and ReplayMod should adapt if those are incompatible (though I think by now there won't be any updates to forge anyway).